java/client/src/main/java/glide/managers/BaseCommandResponseResolver.java

```diff
@@ -27,7 +28,7 @@ public Object apply(Response response) throws RedisException {
         assert !response.hasRequestError() : "Unhandled response request error";
 
         if (response.hasConstantResponse()) {
-            return OK;
+            return GlideString.of(OK);
```

Comment:
Can we store a constant of type `GlidString` instead of creating a new each time?